src/QMCTools/tests/CMakeLists.txt

```diff
@@ -70,4 +70,10 @@ ELSE()
 ENDIF()
 SET_TESTS_PROPERTIES(extract-eshdf-kvectors_2x1x1 PROPERTIES TIMEOUT 120 LABELS "deterministic" PASS_REGULAR_EXPRESSION "0   0   0\n0.5   0   0")
 
-
+# Minimal test for convert4qmc. Check for Usage on output if no arguments. Primarily tested through converter tests.
+IF ( HAVE_MPI )
+  ADD_TEST( NAME convert4qmc_usage COMMAND ${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} 1 ${MPIEXEC_PREFLAGS} ${qmcpack_BINARY_DIR}/bin/convert4qmc )
+ELSE()
+  ADD_TEST(NAME convert4qmc_usage COMMAND "${qmcpack_BINARY_DIR}/bin/convert4qmc")
+ENDIF()
+SET_TESTS_PROPERTIES(convert4qmc_usage PROPERTIES TIMEOUT 120 LABELS "deterministic" PASS_REGULAR_EXPRESSION "Usage:")
```

Comment:
add_unit_test macro as
```
                              PROCESSORS ${TOT_PROCS}
                              ENVIRONMENT OMP_NUM_THREADS=${THREADS}
                              PROCESSOR_AFFINITY TRUE )
```
So it is better to be consistent with other ones.
I think OMP_NUM_THREADS can be just pick magic number 3 if you are not sure whether a test uses OpenMP.
So please add these test properties to this one as well qmc-ckeck-affinity and the one handling supercell (probably all tests recently added) in a separate PR.

Reply:
Incoming

Reply:
Pushed